ansible/roles/schulcloud-server-core/templates/admin-api-server-svc.yml.j2

```
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: admin-api-serv-svc
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: admin-api-serv
+spec:
+  type: ClusterIP
+  ports:
+    # port for http managing drawing data
+    - port: 4030
+      targetPort: 4030
+      protocol: TCP
+      name: admin-api-serv
```

The name of the port shoud be admin 
```suggestion
      name: admin
```

---

done